src/tests/eolian/eolian_generation.c

```diff
@@ -65,17 +65,63 @@ _files_compare (const char *ref_filename, const char *tmp_filename)
    return result;
 }
 
+/**
+ * Returns 0 on success, -1 on failure.
+ */
+static int
+_create_response_file(const char *rsp_path, const char *eo_filename, const char *type, const char *output_filename)
+{
+   remove(rsp_path);
+   FILE *f = fopen(rsp_path, "w");
+
+   if (!f)
+       return -1;
+
+   fprintf(f,
+       "-S\n"
+       "-I\n"
+       ""TESTS_SRC_DIR"\n"
+       "-o\n"
+       "%s:%s\n"
+       "%s",
+       type, output_filename,
+       eo_filename
+   );
```

Comment:
Not an issue but it seems quite odd to me to read, maybe you consider something like the following:
```suggestion
   fprintf(f, "-S\n" "-I\n" TESTS_SRC_DIR "\n" "-o\n" 
             "%s:" "%s\n" "%s",
             type, output_filename, eo_filename);
```

Reply:
I don't see a reason for that (although those last two lines do seems strange). This way it is pretty clear that it is a line by line separation.

Also that reminds me of EFL's leading comma.

Reply:
Yeah, it is ok, it is more a matter of personal taste rather than something technical. Just saying that to me the way is written is not the clearest way to state what is important on that section of code.

Reply:
See if the way I let it now is a little better.